From 3392e772ad63bc8f0a4b9106c31cc3d354f88a73 Mon Sep 17 00:00:00 2001
From: HZHY Device Tree Port <dev@hzhy.com>
Date: Sun, 5 Jul 2026 00:00:00 +0800
Subject: [PATCH 34/39] ARM: dts: rockchip: Makefile: add HZHY MiniEVM SP DTS

Add the HZ-RK3506SP MiniEVM EMMC/NAND board device trees to the build, drop TF variant.
Signed-off-by: HZHY Device Tree Port <dev@hzhy.com>
---
 arch/arm/boot/dts/rockchip/Makefile | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm/boot/dts/rockchip/Makefile b/arch/arm/boot/dts/rockchip/Makefile
index 924ae4bcfa99..109889f32c00 100644
--- a/arch/arm/boot/dts/rockchip/Makefile
+++ b/arch/arm/boot/dts/rockchip/Makefile
@@ -41,3 +41,6 @@ dtb-$(CONFIG_ARCH_ROCKCHIP) += \
 	rk3288-veyron-tiger.dtb \
 	rk3288-vyasa.dtb \
 	rk3506b-aes.dtb \
+	HZ-RK3506SP_MiniEVM_EMMC.dtb \
+	HZ-RK3506SP_MiniEVM_NAND.dtb \
+	HZ-RK3506SP_MiniEVM_SD.dtb
-- 
2.55.0

